src/styles/shared.scss

```diff
@@ -198,7 +198,7 @@ body {
 	}
 }
 .wrapper {
-	padding-top: 50px;
+	padding-top: 100px;
```

Comment:
_Hmmmm_, almost every page have this class _.wrapper_ assigned to them. Would adding an extra 50px to the top padding not affect the rest pages.